Makefile

```diff
@@ -150,6 +150,7 @@ clean: ## Remove all files that are created by building.
 	@$(MAKE) go.mod.clean
 	@$(MAKE) -C images clean
 	@rm -fr $(OUTPUT_DIR) $(WORK_DIR)
+	@rm -fr cluster/olm/ceph/deploy/*
 
```

Comment:
@leseb in .gitignore `cluster/olm/ceph/templates/` is also, do we need to clean that as well?

Reply:
This one gets overwritten, so it won't cause any issues, it won't hurt to remove it too though...